### ISSUE TITLE
Bug in DstToD0pi_D0ToKpipipi BDT selection?

### ISSUE THREAD

**Issue by alexpearce** [alexpearce/CharmProduction#74]
Is this a bug?

```python
BDTselection = ('Dst_ENDVERTEX_CHI2 < 16 && D0_pi_TRACK_CHI2NDOF < 5')
```

Shouldn't it be this?

```patch
diff --git a/analysis/charmproduction/ntuples/DstToD0pi_D0ToKpipipi.py b/analysis/charmproduction/ntuples/DstToD0pi_D0ToKpipipi.py
index 89fb154..12e640b 100644
--- a/analysis/charmproduction/ntuples/DstToD0pi_D0ToKpipipi.py
+++ b/analysis/charmproduction/ntuples/DstToD0pi_D0ToKpipipi.py
@@ -20,7 +20,7 @@ class DstToD0pi_D0ToKpipipi(CharmProductionNtuple):
     depth = D0ToKpipipi.depth
     variables = D0ToKpipipi.variables

-    BDTselection = ('Dst_ENDVERTEX_CHI2 < 16 && D0_pi_TRACK_CHI2NDOF < 5')
+    BDTselection = ('Dst_ENDVERTEX_CHI2 < 16 && Dst_pi_TRACK_CHI2NDOF < 5')

     postselection = ('D0_h1_ProbNNk > 0.1 && D0_h2_ProbNNk < 0.1 && '
                      'D0_h3_ProbNNk < 0.1 && D0_h4_ProbNNk < 0.1')
```

And the [same for `DstToD0pi_D0Tohh`](https://github.com/alexpearce/CharmProduction/blob/0e11821aca73bf81d4eeeed4d8c8b7e3c561afeb/analysis/charmproduction/ntuples/DstToD0pi_D0Tohh.py#L33).

**Comment by chrisburr** [comment]
Yes it is. It must not have been used as it shouldn't even compile that formula.

**Comment by alexpearce** [comment]
I came across it running the analysis script with `--all`.

**Comment by alexpearce** [comment]
I'll make the changes and open a PR.

**Comment by alexpearce** [comment]
Closed by merging #75.
